src/main/java/frc/robot/subsystems/ExampleSubsystem.java

```diff
@@ -19,6 +19,6 @@ public ExampleSubsystem() {
 
   @Override
   public void periodic() {
```

Comment:
This file has nothing to do with anything. I was wondering whether to remove it completely. Don't change it.